site/gdocs/OwidArticlePage.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react"
+
+import { Head } from "../Head.js"
+import { SiteHeader } from "../SiteHeader.js"
+import { SiteFooter } from "../SiteFooter.js"
+import { CitationMeta } from "../CitationMeta.js"
+import { OwidArticle } from "./OwidArticle.js"
+import {
+    OwidArticleType,
+    SiteFooterContext,
+} from "../../clientUtils/owidTypes.js"
+
+declare global {
+    interface Window {
+        _OWID_ARTICLE_PROPS: any
+    }
+}
+
+export default function OwidArticlePage({
+    baseUrl,
+    article,
+}: {
+    baseUrl: string
+    article: OwidArticleType
+}) {
+    const { content, slug, createdAt, updatedAt } = article
+
+    const canonicalUrl = `${baseUrl}/${slug}`
+
+    return (
+        <html>
+            <Head
+                pageTitle={content.title}
+                pageDesc={content.subtitle}
+                canonicalUrl={canonicalUrl}
+                imageUrl={
+                    content["featured-image"]
+                        ? content["featured-image"][0].value.src
```

Comment:
This looks a little unsafe, could we use `_.get(content, ['featured-image', 0, 'value', 'src'], "")` ?